File: PHP/Program/Anggota.php

<!--
    Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP1 dalam mata kuliah
    Desain Pemorograman Berbasis Objek untuk keberkahanNya maka saya tidak 
    melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.
-->

<?php

//membuat kelas dengan nama anggota
class Anggota{
    //atribut privat
    private $id = '';
    private $nama = '';
    private $bidang = '';
    private $partai = '';
    private $foto = '';

    //Constructor
    //Atribut publik
    public function __construct($id = '', $nama = '', $bidang = '', $partai = '', $foto = ''){
        //set atribut dengan parameter
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->foto = $foto;
    }

    // Getter dan Setter
    
    public function setId($id){
        $this->id = $id;
    }

    public function getId(){
        return $this->id;
    }

    public function setNama($nama){
        $this->nama = $nama;
    }

    public function getNama(){
        return $this->nama;
    }

    public function setBidang($bidang){
        $this->bidang = $bidang;
    }

    public function getBidang(){
        return $this->bidang;
    }

    public function setPartai($partai){
        $this->partai = $partai;
    }

    public function getPartai(){
        return $this->partai;
    }

    public function setFoto($foto){
        $this->foto = $foto;
    }

    public function getFoto(){
        return $this->foto;
    }


}

?>

File: Python/Program/Anggota.py
## Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP1 dalam mata kuliah 
## Desain Pemorograman Berbasis Objek untuk keberkahanNya maka saya tidak 
## melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.

class Anggota:
    
    ## atribut privat
     __id = ""
     __nama = ""
     __bidang = ""
     __partai = ""
     
     ##Constructor
     
     ##atribut publik
     def __init__(self, id = "", nama = "", bidang = "", partai = ""):
         ##set atribut dengan parameter
         self.__id = id
         self.__nama = nama
         self.__bidang = bidang
         self.__partai = partai
         
     ##Getter dan Setter
     
     # Get id    
     def get_id(self):
         return self.__id
     
     # Set id
     def set_id(self, id):
         self.__id = id
     
     # Get nama       
     def get_nama(self):
         return self.__nama
     
     # Set nama
     def set_nama(self, nama):
         self.__nama = nama
     
     # Get bidang    
     def get_bidang(self):
         return self.__bidang
     
     # Set nama
     def set_bidang(self, bidang):
         self.__bidang = bidang
     
      # Get partai     
     def get_partai(self):
         return self.__partai
     
     # Set partai
     def set_partai(self, partai):
         self.__partai = partai
         
    
   
    
  